### PR TITLE
[CDBT] - Template Updates

### PR DESCRIPTION
**Pull request by jack-burt-is**
Covers bug fixes and enhancements including:
- [[CDBT-185] - Split Applications Email Templates](https://dsdmoj.atlassian.net/browse/CDBT-185)
- [[CDBT-149] - Multiple CRN Confirmation](https://dsdmoj.atlassian.net/browse/CDBT-149)
- [[CDBT-201] - Hide under 18 on Summary](https://dsdmoj.atlassian.net/browse/CDBT-201)
- [[CDBT-298] - Fix email templates for bereavement claims](https://dsdmoj.atlassian.net/browse/CDBT-298)

### CODE REVIEW

**Comment by neil-stephen-mcgonigle** [comment]
Jack are these changes backwards compatible with data capture service?
image

File: src/lib/resource/sections/applicant-are-you-18-or-over.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = {
                                         myself: 'Are you 18 or over?',
                                         proxy: 'Are they 18 or over?'
                                     }
-                                }
+                                },
```
Comment (neil-stephen-mcgonigle) [inline src/lib/resource/sections/applicant-are-you-18-or-over.js:30]: unnecessary comma

**Comment by jack-burt-is** [comment]
> @jack-burt-is are these changes backwards compatible with data capture service? image

They should all be backwards compatible yes, so a minor version is likely most appropriate

**Review by neil-stephen-mcgonigle — CHANGES_REQUESTED** [review]

**Review by neil-stephen-mcgonigle — APPROVED** [review]

**Comment by neil-stephen-mcgonigle** [comment]
@jack-burt-is there are failing tests.

**Review by neil-stephen-mcgonigle — CHANGES_REQUESTED** [review]
Failing tests

**Comment by neil-stephen-mcgonigle** [comment]
- already merged to master

**Pull request closed without merging**